packages/melos/lib/src/command_runner/base.dart

```diff
@@ -12,8 +13,16 @@ abstract class MelosCommand extends Command<void> {
 
   final MelosWorkspaceConfig config;
 
-  Logger get logger =>
-      globalResults!['verbose'] as bool ? Logger.verbose() : Logger.standard();
+  Logger get logger {
+    if (currentPlatform.environment['MELON_VERBOSE'] != null) {
```

Comment:
```suggestion
    if (currentPlatform.environment['MELOS_VERBOSE'] != null) {
```

Please make a const for it like these ones: https://github.com/invertase/melos/blob/main/packages/melos/lib/src/common/utils.dart#L60-L62

It will avoid typos then 😅